src/Packagist/WebBundle/Tests/Package/SymlinkDumperTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Packagist\WebBundle\Tests\Package;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Packagist\WebBundle\Package\SymlinkDumper;
+
+class SymlinkDumperTest extends WebTestCase
+{
+    protected static $kernel;
+
+    protected static $dumper;
+
+    public static function setUpBeforeClass()
+    {
+         self::$kernel = static::createKernel();
+         self::$kernel->boot();
+
+         $container = self::$kernel->getContainer();
+         self::$dumper = $container->get('packagist.package_dumper');
```

Comment:
instead of booting a kernel here and getting the full object graph, you should rather create mocks for dependencies to make it a unit test (your tested code does not involve any of the dependencies anyway)